scripts/Docker/readme.md

```diff
@@ -0,0 +1,76 @@
+# Librealsense Docker
+
+## Pre-Work: Docker Installation
+Install docker in the environemnt using the  following tutorial (adjust the OS):
```

Comment:
typo in environment
.."adjust **for** the OS

Reply:
Done